src/screens/Explore/index.tsx

```diff
@@ -66,7 +66,7 @@ export default function Explore() {
     }, 100);
   };
   const categoryListComponent = () => {
-    const categoryElements = [];
+    const categoryElements: JSX.Element[] = [];
```

Comment:
can we refactor this because imo, we should not use array of elements

Reply:
refactored Done !